odom_frame        : "odom"
base_frame        : "base_link"
global_frame      : "map"
laser_topic_name     : "scan"
map_topic_name       : "map"
init_pose_topic_name : "initialpose"
transform_tolerance  : 0.1
#8.40 #1.0
#14.35 #1.0
initial_pose_x : 1
initial_pose_y : 1 
initial_pose_a : 3.14
initial_cov_xx : 0.1
initial_cov_yy : 0.1
initial_cov_aa : 0.1

enable_uwb : false
uwb_frame_id : "uwb"
uwb_topic_name : "uwb"
use_sim_uwb : false
uwb_correction_frequency : 20
